conf/adcm/python_scripts/run_ldap_sync.py

```diff
@@ -114,7 +118,7 @@ def _sync_ldap_groups(self, ldap_groups):
         django_groups = set(Group.objects.filter(type=OriginType.LDAP).values_list('display_name', flat=True))
         for groupname in django_groups - new_groups:
             group = Group.objects.get(name__iexact=f'{groupname} [ldap]')
-            sys.stdout.write(f"We will delete this group: {group}\n")
+            sys.stdout.write(f"Delete this group: {group}\n")
             group.delete()
         msg = "Sync of groups ended successfully."
         msg += f"Couldn\'t synchronize groups: {error_names}\n" if error_names else "\n"
```

Comment:
need to remove \n here  (because logger go to new line itself)
This is right variant: 
msg += f"Couldn\'t synchronize groups: {error_names}" if error_names else ""